### PR TITLE
 Updating all unit test assemblies to use xunit.runner.json instead of… 

### PR DESCRIPTION
**Pull request by tannergooding**
… app.config files.

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @jaredpar. This is part two of the change and is dependent on #6087

File: src/Compilers/CSharp/Test/CommandLine/xunit.runner.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "parallelizeAssembly": true,
```
Comment (jaredpar) [inline src/Compilers/CSharp/Test/CommandLine/xunit.runner.json]: What are the defaults for this file?  I think we need to ensure parallelizeCollection is false.  I'd personally rather be explicit htan implicit.

Comment (tannergooding) [inline src/Compilers/CSharp/Test/CommandLine/xunit.runner.json]: Updated to be explicit.

**Comment by tannergooding** [comment]
test win please

**Comment by tmeschter** [comment]
Since all of these files are named xunit.runner.json, aren't they going to be clobbering each other in the output directory during the build?

**Comment by davkean** [comment]
I don't think we want to take this for two reasons; we're using binding redirects in the App.Config, which cannot be represented in this file, and second for Tom's comment above, we have different settings across projects. These will overwrite each other. App.Config when deployed to the output directory takes is prefixed with the name of the assembly.

Given this is now stale, I'll close. Feel free to reopen if you want to argue a case for it.

**Pull request closed without merging**